AEPCore/Sources/eventhub/EventSource.swift

```diff
@@ -36,4 +36,8 @@ public class EventSource: NSObject {
     public static let applicationClose = "com.adobe.eventSource.applicationClose"
     public static let personalizationDecisions = "personalization:decisions"
     public static let locationHintResult = "locationHint:result"
+    public static let errorResponseContent = "com.adobe.eventSource.errorResponseContent";
+    public static let requestTracker = "com.adobe.eventSource.requestTracker";
```

Comment:
nit: Since media doesn't send a corresponding response event, should we use createTracker instead?